wolox-books/src/app/unauth.guard.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, Router } from '@angular/router';
+import { UserService } from './services/user.service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UnauthGuard implements CanActivate {
+
+  constructor(
+    private UserService: UserService,
+    private router: Router
+  ) {}
+
+  canActivate(): boolean {
+    if (this.UserService.loggedIn() == false ) {
```

Comment:
use:
```
if (!this.UserService.loggedIn()) {
```

and NEVER use `==`, always use `===` since the later one also compares types